sensors/ev3_uart_sensor_defs.c

```diff
@@ -46,7 +46,7 @@ const struct ev3_uart_sensor_info ev3_uart_sensor_defs[] = {
 			},
 			[1] = {
 				/**
-				 * @description: Ambient light<br />Red LEDs off
+				 * @description: Ambient light<br />Blue LEDs on
```

Comment:
Are you sure? The brightness seems to be the same as of the red light. And there is no other mode with blue color... Except modes with white, which is combination of dimmed rgb (I'm guessing).